x-pack/legacy/plugins/beats_management/server/lib/adapters/configuration_blocks/elasticsearch_configuration_block_adapter.ts

```diff
@@ -137,12 +137,13 @@ export class ElasticsearchConfigurationBlockAdapter implements ConfigurationBloc
   public async create(user: FrameworkUser, configs: ConfigurationBlock[]): Promise<string[]> {
     const body = flatten(
       configs.map((config) => {
-        const id = config.id || uuidv4();
+        const { id: configId, ...configWithoutId } = config;
```

Comment:
I think you can also do `id = uuidv4()` to use uuid if id is falsy. No issue, just an option.

Reply:
@nchaulet Do you want to accept this suggestion? Or can I merge the PR?

Reply:
I am currently in a phone I think you can merge it as it is.